rhino/src/test/java/org/mozilla/javascript/tests/ArrowFunctionToStringTest.java

```diff
@@ -0,0 +1,12 @@
+package org.mozilla.javascript.tests;
+
+import org.junit.jupiter.api.Test;
+import org.mozilla.javascript.testutils.Utils;
+
+public class ArrowFunctionToStringTest {
+    @Test
+    public void EvalArrowToString() {
```

Comment:
lowercase at the beginning of method names in java ;-)

Reply:
Will address! Thank you.